captum/concept/__init__.py

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python3
+
+__version__ = "0.3.0"
```

Comment:
Do we need version specified in this package init? Can look into this in a separate diff once this lands though.

Reply:
Good point. I'm planning to remove it in the next PR.